app/views/static_pages/help.html.erb

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title><%=t ".title"%><%=t "main_title"%></title>
```

Comment:
`<%=t ".title"%><%=t "main_title"%>
sửa lại
<%= t ".title" %><%= t "main_title" %>

Sefl check lại em nhé

2. Dùng I18n cho toàn bộ file nhé